framework/src/modules/chain/blocks/blocks.js

```diff
@@ -452,21 +488,20 @@ class Blocks extends EventEmitter {
 
 	async _receiveBlockFromNetworkV2(block) {
 		// Current time since Lisk Epoch
-		// Better to do it here rather than in the Sequence so receiving time is more accurate
-		const newBlockReceivedAt = this.slots.getTime();
+		this._lastReceipt = Math.floor(this.slots.getTime() / 1000);
 
 		// Execute in sequence
 		return this.sequence.add(async () => {
 			this._shouldNotBeActive();
 			this._isActive = true;
-			const result = await this._forkChoiceTask(block, newBlockReceivedAt);
+			const result = await this._forkChoiceTask(block);
 			this._isActive = false;
```

Comment:
forkChoiceTask should declare finally block which sets the isActive property to true, as we discussed.